ydb/core/mind/local.cpp

```diff
@@ -278,15 +281,35 @@ class TLocalNodeRegistrar : public TActorBootstrapped<TLocalNodeRegistrar> {
         auto& record = eventStatus->Record;
         record.SetStartTime(StartTime.GetValue());
         record.MutableResourceMaximum()->CopyFrom(ResourceLimit);
+        std::vector<NActors::TExecutorPoolState> poolStates;
+        ctx.ExecutorThread.ActorSystem->GetExecutorPoolStates(poolStates);
+
+        TDuration passedTime = ctx.Now() - LastUpdate;
+        LastUpdate = ctx.Now();
+
+        auto *actorSystemInfo = record.MutableActorSystemInfo();
+        double cores = 0;
+        for (ui8 poolId = 0; poolId < poolStates.size(); ++poolId) {
+            auto &poolState = poolStates[poolId];
+            if (poolId != AppData()->IOPoolId) {
```

Comment:
We already have a setting PoolsToMonitorForUsage. I believe it should be used here. But it also excludes Batch pool by default - is there a reason you're including it here?